src/lifecycle/properties-ready.js

```diff
@@ -1,6 +1,5 @@
 export default function propertiesApply (elem, properties) {
   Object.keys(properties).forEach(function (name) {
-    let prop = properties[name];
-    prop.set && prop.set.call(elem, elem[name]);
+    properties[name].init.call(elem);
```

Comment:
In the `init` function, it does call set. This `init` function isn't part of public API.